unattended_installer/install_functions/installCommon.sh

```diff
@@ -267,10 +275,10 @@ function installCommon_installPrerequisites() {
         not_installed=()
         for dep in "${dependencies[@]}"; do
             if [ "${dep}" == "openssl" ]; then
-                if ! yum list installed 2>/dev/null | grep -q "${dep}\.";then
+                if ! yum list installed 2>/dev/null | grep -q -E ^"${dep}\.";then
                     not_installed+=("${dep}")
                 fi
-            elif ! yum list installed 2>/dev/null | grep -q "${dep}";then
+            elif ! yum list installed 2>/dev/null | grep -q -E ^"${dep}";then
```

Comment:
What is the reason for this change?

Reply:
In a conversation with @DFolchA we noticed that the `grep` command may fetch some undesirable packages. With this change, we ensure that we are fetching the packages we need, specifying that its name must be at the beginning of the package name.